services/src/unit-test/java/uk/gov/moj/sdt/services/SubmitQueryServiceTest.java

```diff
@@ -121,6 +106,14 @@ public void setUp() {
                                                     mockBulkCustomerDao);
         submitQueryService.setBulkCustomerDao(mockBulkCustomerDao);
 
+        //To cover code coverage
+        submitQueryService.setErrorMessagesCache(mockErrorMsgCacheable);
+        submitQueryService.setGlobalParametersCache(mockGlobalParamCache);
+        submitQueryService.setQueryResponseXmlParser(genericParser);
+        submitQueryService.setQueryRequestXmlParser(genericParser);
+        submitQueryService.setRequestConsumer(mockConsumerGateway);
```

Comment:
As much as I hate to say it, these set methods need to be separated out into individual tests.  Using both them and the constructor to initialise the object means we don't know whether the constructor works or not.

Reply:
done